### DISCUSSION TITLE
ArgoCD Finalizers

### DISCUSSION THREAD

**Discussion by ghost** [argoproj/argo-cd#18647] [Q&A]
Hi Argo CD community,
I have a doubt about finalizers on argo cd, on my argo cd instance I have an app which uses an CRD on a helm chart , so every time I delete the app my CRD still applied to my cluster so the side effect is that my operator and argo cd enter in a death lock since my operator stills applying the K8s objects but argo cd keeps deleting that objects created by the operator.
After inspecting the CRD I notice that on the metadata it was added the finalizer `foregroundDeletion` so I was wondering if there is any way to configure on argo cd not to add that finalizer to the CRD of maybe other kind of delete policy should be configured so every time my app is deleted deletes the objects defined on the CRD as well it deletes it so the operator doesnt continue with that behaviour.
Thanks in advance.

**Comment by rumstead** [comment]
See comments on the ticket: https://github.com/argoproj/argo-cd/issues/18672
